test/test.js

```diff
@@ -495,6 +510,44 @@ tape('Non-JSON logs may carry timestamp.', function (t) {
   logger[lvl]('test');
 });
 
+tape('Non-JSON logs may carry Hostname.', function (t) {
+  t.plan(1);
+  t.timeoutAfter(2000);
+
+  mockTest(function (data) {
+    t.true(pattern.test(data), 'matched');
+
+  });
+  const os = require('os');
+  const lvl = defaults.levels[3];
+  const tkn = token;
+  const pattern = new RegExp('^' + token +' ' + os.hostname() + ' \\w+ test\\n$'
```

Comment:
would this be more readable with template strings?
```suggestion
  const pattern = new RegExp(`^${token} ${os.hostname()} \\w+ test\\n$`;
```